include/seqan3/std/span

```diff
@@ -86,8 +86,9 @@ public:
     constexpr span(iterator_t f, index_type count) noexcept : data_{addressof(*f)}
         { (void) count; assert(span_extent == count); } // "size mismatch in span's constructor (f, count)"
 
-    template <contiguous_iterator iterator_t, sized_sentinel_for<iterator_t> sentinel_t>
-        requires is_convertible_v<remove_reference_t<iter_reference_t<iterator_t>>(*)[], element_type(*)[]> &&
+    template <contiguous_iterator iterator_t, typename sentinel_t>
+        requires sized_sentinel_for<sentinel_t, iterator_t> &&
```

Comment:
doxugen `\\!cond` missing for all of them

Reply:
can't do this, because the complete file is `//!cond`; see line17 to 425